upgrade/overview.md

```diff
@@ -1,54 +1,44 @@
-* [Upgrade Manually Added Assemblies](#upgrade-manually-added-assemblies)
-* [Upgrade from Trial to Commercial](#upgrade-from-trial-to-commercial)
-* [Troubleshooting](#troubleshooting)
+* <a href="https://www.telerik.com/support/whats-new/maui-ui/release-history" target="_blank">UI for .NET MAUI release history</a>&mdash;Reviewing the release notes for all releases you're planning to go through reveals all changes, fixes, new features, and components that have been implemented.
+* [UI for .NET MAUI breaking changes]({%slug versions-with-breaking-changes%})&mdash;Reviewing the breaking changes helps you prepare for any additional steps that you may need to perform. The Breaking Changes article lists the releases that introduce a breaking change and helps you evaluate if your project will be affected by such a change.
 
 ## Upgrade Using the Telerik NuGet Server
 
-To upgrade the Telerik UI for .NET MAUI components used in your project, perform the following steps:
+To upgrade the Telerik UI for .NET MAUI components used in your project:
 
-1. Make sure that you have a NuGet feed source with the version you want to upgrade to. This is usually the Telerik NuGet Feed&mdash;on [Windows]({%slug telerik-nuget-server%}) or [MacOS]({%slug telerik-nuget-server-mac%}), but you can also use a local feed from our [MSI installer]({%slug maui-getting-started%}) or [PKG installer]({%slug install-pkg%}) or [ZIP archive]({%slug download-product-files%}).
+1. Make sure that you have a NuGet feed source with the version you want to upgrade to. This is usually the Telerik NuGet Feed (on [Windows]({%slug telerik-nuget-server%}) or [MacOS]({%slug telerik-nuget-server-mac%})), but you can also use a local feed from the [MSI installer]({%slug maui-getting-started%}) or [PKG installer]({%slug install-pkg%}) or [ZIP archive]({%slug download-product-files%}).
 
 1. Update the version of the `Telerik.UI.for.Maui` package your project references. If you are using a trial version, the package name is `Telerik.UI.for.Maui.Trial`.
 
 ## Upgrade Manually Added Assemblies
 
-To upgrade your components to a newer version of the suite, you need to perform the instructions described in the [Manually Using Assembly References]({%slug installation-approaches%}#manually-using-assembly-references) article.
+You can use a [manual installation approach]({%slug installation-approaches%}#manually-using-assembly-references) for referencing the required Telerik UI for .NET MAUI assemblies into your solution. In this case, to upgrade the components to a newer version, you need to replace the assemblies with the assemblies from the latest released version of Telerik .NET MAUI.
 
-> If you have installed the trial version of Telerik UI for .NET MAUI and want to install the developer version, you have to remove the trial version first.
+> If you have installed the trial version of Telerik UI for .NET MAUI and want to install the commercial version, you have to remove the trial version first.
```

Comment:
If this is a requirement, it shouldn't be a note in this section. It should be the first step in the next section that handles the upgrade from trial to commercial.